src/Core/src/Core/ISwipeView.cs

```diff
@@ -40,10 +40,22 @@ public interface ISwipeView : IContentView
 		/// </summary>
 		public SwipeTransitionMode SwipeTransitionMode { get; }
 
+		/// <summary>
+		/// Event that is fired when the swipe starts.
```

Comment:
It's a method, not an event (in the C# sense) - maybe instead of "Event that is fired when X", just say "Called when X" for these.